pkg/antctl/raw/multicluster/create/clusterset.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type clusterSetOptions struct {
+	leaderCluster          string
+	leaderClusterServer    string
+	leaderClusterNamespace string
+	memberClusters         map[string]string
+	namespace              string
+	secret                 string
+}
+
+var clusterSetOpt *clusterSetOptions
+
+var clusterSetExamples = strings.Trim(`
+Create a ClusterSet in a leader or member cluster
+$ antctl mc create clusterset <CLUSTERSET> -n <NAMESPACE> --leader-server <LEADER_SERVER>  --service-account <SERVICE_ACCOUNT> --secret <SECRET> --leader-cluster <LEADER_CLUSTER_ID>
+$ antctl mc create clusterset test-clusterset -n antrea-mc-test --leader-server https://127.0.0.1 --member-clusters member1=sa1,member2=sa2 --secret test-secret --leader-namespace test-leader-ns --leader-cluster-id example-id
```

Comment:
Maybe add a port in the sample so user can learn what's the exact format it requires
```suggestion
$ antctl mc create clusterset test-clusterset -n antrea-mc-test --leader-server https://127.0.0.1:6443 --member-clusters member1=sa1,member2=sa2 --secret test-secret --leader-namespace test-leader-ns --leader-cluster-id example-id
```

Reply:
Done